djmoney/models/fields.py

```diff
@@ -34,12 +34,17 @@
 if 'djmoney_rates' in settings.INSTALLED_APPS:
     try:
         from djmoney_rates.utils import convert_money
```

Comment:
IMO it is a bad thing - to import models on module level. In some cases it leads to "app registry is not ready" errors.